website/docs/r/datahub_subscription.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_datahub_subscription"
+sidebar_current: "docs-alicloud-resource-datahub-subscription"
+description: |-
+  Provides a Alicloud datahub subscription resource.
+---
+
+# alicloud\_datahub\_subscription
+
+The subscription is the basic unit of resource usage in Datahub Service under Publish/Subscribe model. You can manage the relationships between user and topics by using subscriptions. [Refer to details](https://help.aliyun.com/document_detail/47440.html).
+
+## Example Usage
+
+Basic Usage
+
+```
+resource "alicloud_datahub_subscription" "example" {
+  project_name = "tf_datahub_project"
+  topic_name = "tf_datahub_topic"
+  comment = "created by terraform"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project_name` - (Required, ForceNew) The name of the datahub project that the subscription belongs to. Its length is limited to 3-32 and only characters such as letters, digits and '_' are allowed. It is case-insensitive.
+* `topic_name` - (Required, ForceNew) The name of the datahub topic that the subscription belongs to. Its length is limited to 1-128 and only characters such as letters, digits and '_' are allowed. It is case-insensitive.
+* `comment` - (Optional) Comment of the datahub subscription. It cannot be longer than 255 characters.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of the datahub subscritpion as terraform resource. It was composed of project name, topic name and practical subscription ID generated from server side.
```

Comment:
Please show the id format. `The ID of the datahub subscritpion as terraform resource. It is composed of project name, topic name and practical subscription ID generated from server side. Format to <project_name>:<name>:<sub_id> .